docs/book/src/developers/releasing.md

```diff
@@ -164,15 +167,6 @@ Consider whether anything should be updated in the [roadmap document](../roadmap
 
 If any changes need to be made, it should not block the release itself.
 
-#### Patch Releases
-
-1. Announce the release in Kubernetes Slack on the #cluster-api-azure channel.
-
-#### Minor/Major Releases
-
-1. Follow the communications process for [pre-releases](#pre-releases)
```

Comment:
now that I'm re-reading this I think this was supposed to be 

Follow the communications process for [patch releases](#patch-releases)

Reply:
or maybe it could just be 

```
1. Announce the release in Kubernetes Slack on the #cluster-api-azure channel.
```